app/controllers/questionnaires_controller.rb

```diff
@@ -363,6 +363,31 @@ def valid_quiz
     valid
   end
 
+  # Methods for Supplementary Review Questionnaire
+
+  def create_srq
+    @participant = AssignmentParticipant.find(params[:id])
+    @team = Team.find(@participant.team.id)
+    if @team.srq_id.nil? then
+      @questionnaire = Questionnaire.new
```

Comment:
Questionnaire creation code can use `create` instead. And it is unnecessary to use the instance variable. Also, you can make values as parameters, instead of assigning them one by one.